Makefile

```diff
@@ -53,3 +70,34 @@ release: build
 	@zip -r $(RELEASE_FILENAME).zip $(RELEASE_FILENAME)
 	@tar -czvf $(RELEASE_FILENAME).tar.gz $(RELEASE_FILENAME)
 	@rm -rf $(RELEASE_FILENAME)
+
+$(VERSION_FILE): # Creates $(VERSION_FILE) file
+	@echo "GIT_BRANCH \"`git rev-parse --abbrev-ref HEAD`\"" > $(VERSION_FILE)
+	@echo "COMMIT_HASH \"`git rev-parse --short HEAD`\"" >> $(VERSION_FILE)
+	@echo "BUILD_DATE_UTC \"`date -u +"%F %T %Z"`\"" >> $(VERSION_FILE)
+
+$(VENV_PATH): build
+	$(PYTHON_COMMAND)$(PYTHON_VER) -m venv $(VENV_PATH)
+	$(VENV_BIN)/$(PYTHON_COMMAND) -m $(PIP_COMMAND) install --upgrade $(PIP_COMMAND)
+
+$(VENV_PATH)/req-done: $(VENV_PATH) requirements.txt
+	$(VENV_BIN)/$(PYTHON_COMMAND) -m $(PIP_COMMAND) install -r requirements-dev.txt $(PYPI_CMD)
+	$(VENV_BIN)/$(PYTHON_COMMAND) -m $(PIP_COMMAND) download ottopia-logging $(PYPI_CMD)
```

Comment:
I think the order should be first download then install

Reply:
the download is used later by the make install. the install is for development purposes. i agree it should be more synchronised. we'll do so later, wdyt?